scripts/presubmit-tests.sh

```diff
@@ -368,17 +369,19 @@ function main() {
       header "Documentation only PR, skipping running custom test"
       exit 0
     fi
-    ${TEST_TO_RUN} || failed=1
+    for test_to_run in ${TESTS_TO_RUN[@]}; do
```

Comment:
Could you update the document for this change?

Reply:
added